src/node_native_module.cc

```diff
@@ -174,6 +174,48 @@ MaybeLocal<Function> NativeModuleLoader::CompileAsModule(
   return LookupAndCompile(context, id, &parameters, result);
 }
 
+Local<String> NativeModuleLoader::LoadInternalModuleSource(Isolate* isolate,
+                                                           const char* id) {
+#ifdef NODE_LOAD_INTERNAL_MODULES_FROM_DISK
+    // Filename on disk
+    std::string filename = "./lib/";
+    filename += id;
+    filename += ".js";
+
+    uv_fs_t req;
+    uv_file file =
```

Comment:
This file descriptor is leaked, there's no corresponding call to `uv_fs_close()`.